packages/core/src/env-set/index.ts

```diff
@@ -63,7 +63,7 @@ export class EnvSet {
     return this.#oidc;
   }
 
-  async load() {
+  async load(customDomain?: string) {
```

Comment:
actually there's no need to separate default and custom domains? just pass an endpoint and use it as the cache key and issuer.

Reply:
Then path based multi tenancy will break.

Reply:
discussed offline